setup.py

```diff
@@ -435,7 +435,7 @@ def do_setup():
         long_description_content_type='text/markdown',
         license='Apache License 2.0',
         version=version,
-        packages=find_packages(exclude=['tests*']),
+        packages=find_namespace_packages(exclude=['tests*']),
```

Comment:
Is it a related change?

Reply:
Havn't finished yet, but yest - that's the way to find packages when doing "pip install". We are using Native namespace packages as described here: https://packaging.python.org/guides/packaging-namespace-packages/#native-namespace-packages